src/pages/information/map/CreateResource.js

```diff
@@ -56,11 +59,54 @@ export default function CreateResource() {
         navigate("/information/admin");
       })
       .catch((error) => {
-        console.log(error);
-        swal(errorMsg);
+         if (error.response) {
+          let error_msgs = {general: "No ha rellenado correctamente."};
+          setErrors(error_msgs);
+        }else { 
+          swal(errorMsg);
+        }
```

Comment:
```suggestion
        } 
         swal(errorMsg);
        
```
Así, cuando no se encuentra la dirección, salta el error general. Que antes no daba nada de feedback al usuario.